cvmfs/catalog.h

```diff
@@ -190,6 +190,12 @@ class Catalog : public SingleCopy {
     return inode_range_.IsInitialized() && initialized_;
   }
   inline bool IsRoot() const { return is_root_; }
+  bool IsAutogenerated() const {
+    DirectoryEntry dirent;
+    return IsInitialized() &&
```

Comment:
Hmmm it could be, but I thought it would make sense to leave it this way. A catalog that is not initialised doesn't know if it is autogenerated or not, so in principle we suppose it isn't